src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -48,6 +47,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.LogManager;
```

Comment:
unused import AFAICT? (I'm surprised the linter doesn't catch this)

Reply:
Interestingly, I ran `checkstyle` linter locally and it also didn't see this line as a problem.